pages/linux/clamav.md

```diff
@@ -0,0 +1,21 @@
+# clamav
+
+> Clam AntiVirus is an open source anti-virus.
+> Designed especially for e-mail scanning on mail gateways.
```

Comment:
Perhaps this will read better?
```suggestion
> Open-source anti-virus program.
> Designed especially for e-mail scanning on mail gateways, but can be used in other contexts.
```